SQL: Adapt the limitations page to the new "fields" API usage (#69616)
elastic/elasticsearch: docs/reference/sql/limitations.asciidoc

Therefore calling `ST_Z` function in the filtering, grouping or sorting will return `null`.

[discrete]
[[using-fields-api]]

> === Retrieving using "fields" search parameter

Reviewer: ```suggestion
=== Retrieving using the `fields` search parameter
```

> {es-sql} retrieves columns' values using <<search-fields-param,`_search` "fields" parameter>> and, as such, any
> limitations this parameter has, will be visible in {es-sql} queries, as well. For example, if `_source` is disabled

Reviewer: ```suggestion
{es-sql} retrieves column values using the <<search-fields-param,search API's
`fields` parameter>>. Any limitations on the `fields` parameter also apply to
{es-sql} queries. For example, if `_source` is disabled
```
